proto/viam/app/v1/billing.proto

```diff
@@ -62,9 +62,39 @@ message GetCurrentMonthUsageRequest {
   string org_id = 1;
 }
 
+enum UsageCostType {
+  CLOUD_STORAGE = 1; 
+  DATA_UPLOAD = 2; 
+  DATA_EGRESS = 3;
+  REMOTE_CONTROL = 4; 
+  STANDARD_COMPUTE = 5; 
+  BINARY_DATA_CLOUD_STORAGE = 6; 
+  OTHER_CLOUD_STORAGE = 7;
+  PER_MACHINE = 8;
+}
+
+message UsageCost {
```

Comment:
this is to replace the old way of how we did this which was just list the costs directly on the struct. We want something like this so its usable for fragments too